src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable_factory.ts

```diff
@@ -73,16 +70,18 @@ export class SearchEmbeddableFactory extends EmbeddableFactory<
     input: Partial<SearchInput> & { id: string; timeRange: TimeRange },
     parent?: Container
   ): Promise<SearchEmbeddable | ErrorEmbeddable> {
-    const $injector = await chrome.dangerouslyGetActiveInjector();
+    const $injector = await getServices().chromeLegacy.dangerouslyGetActiveInjector();
 
     const $compile = $injector.get<ng.ICompileService>('$compile');
     const $rootScope = $injector.get<ng.IRootScopeService>('$rootScope');
     const searchLoader = $injector.get<SavedSearchLoader>('savedSearches');
-    const editUrl = chrome.addBasePath(`/app/kibana${searchLoader.urlFor(savedObjectId)}`);
+    const editUrl = await getServices().chromeLegacy.addBasePath(
```

Comment:
This can be done without `chromeLegacy`, the function is now on `core.http.basePath.prepend`

Reply:
cool thx!

Reply:
Refactored this part